src/resolver.js

```diff
@@ -0,0 +1,102 @@
+'use strict'
+
+const util = require('./util')
+
+const resolve = (binaryBlob, path, callback) => {
```

Comment:
nitpick, you could use `function` declarations here instead of assigning funcs to const.

Reply:
I personally prefer `var functionname = function(bla) {}` style over `function functioname(bla)`. But if the general style is `function functioname(bla)` I'm happy to change that in the whole code base.

Reply:
Right now we don't have a fixed style although I do write and push more for the `function name ()`. Let's keep it as it is then.